variables.tf

```diff
@@ -22,28 +15,63 @@ variable "resource_group_name" {
   description = "The resource group where the resources will be deployed."
 }
 
+
+# New variables for APIM
+variable "publisher_name" {
+  type        = string
+  description = "The name of the API Management service publisher."
+  default     = "Apim Example Publisher"
+}
+
+variable "publisher_email" {
+  type        = string
+  description = "The email of the API Management service publisher."
+  default     = ""
```

Comment:
should we add a `validation` block here? if I remember correctly, I don't think we can deploy APIM without setting a publisher email?